### PR TITLE
🌉New "mobile" view for ads in fullbleed story

### PR DESCRIPTION
**Pull request by calebcordry**
image

Story ads so far have not been responsive. Introduces a new view for fullbleed stories on desktop where we render the ad in a mobile like size.

### CODE REVIEW

File: extensions/amp-story-auto-ads/0.1/story-ad-page.js [hunk 1]
```diff
@@ -127,6 +139,9 @@ export class StoryAdPage {
 
     /** @private {boolean} */
     this.viewed_ = false;
+
+    /** @private @const {!../../amp-story/0.1/amp-story-store-service.AmpStoryStoreService|!../../amp-story/1.0/amp-story-store-service.AmpStoryStoreService} */
```
Comment (powerivq) [inline extensions/amp-story-auto-ads/0.1/story-ad-page.js]: Is it desirable to have inter-version dependency?

Comment (gmajoulet) [reply extensions/amp-story-auto-ads/0.1/story-ad-page.js]: Good point! We should clean it but that'd be on the AMP Story WG to do it when we delete all the `0.1` code

File: extensions/amp-story-auto-ads/0.1/story-ad-page.js [hunk 2]
```diff
@@ -48,6 +53,9 @@ const TIMEOUT_LIMIT = 10000; // 10 seconds
 /** @const {string} */
 const GLASS_PANE_CLASS = 'i-amphtml-glass-pane';
 
+/** @const {string} */
+const DESKTOP_FULLBLEED_CLASS = 'i-amphtml-story-desktop-fullbleed';
```
Comment (gmajoulet) [inline extensions/amp-story-auto-ads/0.1/story-ad-page.js]: Should we add another prefix to make it different from the class the AMP Story runtime uses? I'm afraid of CSS properties collisions eventually

File: extensions/amp-story-auto-ads/0.1/story-ad-page.js [hunk 3]
```diff
@@ -480,24 +502,49 @@ export class StoryAdPage {
...
+  onUIStateUpdate_(uiState) {
+    if (!this.adChoicesIcon_) {
+      return;
+    }
+
+    this.adChoicesIcon_.classList.remove(DESKTOP_FULLBLEED_CLASS);
+
+    if (uiState === UIType.DESKTOP_FULLBLEED) {
+      this.adChoicesIcon_.classList.add(DESKTOP_FULLBLEED_CLASS);
+    }
```
Comment (gmajoulet) [inline extensions/amp-story-auto-ads/0.1/story-ad-page.js]: If you feel like factorizing this in a single line (it's a bit less clear tho, so up to you)
```
this.adChoicesIcon_.classList.toggle(
    DESKTOP_FULLBLEED_CLASS, uiState === UIType.DESKTOP_FULLBLEED);
```

**Pull request merged**